src/components/checkbox-drawer/checkbox-drawer.js

```diff
@@ -64,17 +53,14 @@ class CheckboxDrawer extends LocalizeLabsElement(SkeletonMixin(LitElement)) {
 					class="d2l-input-checkbox"
 					?skeleton="${this.skeleton}"
 					description=${this.localize(`components:checkboxDrawer:checkbox${this.checked ? 'Expanded' : 'Collapsed'}`)}
-					@change="${this._onCheckboxChange}">${this.label}</d2l-input-checkbox>
+					@change="${this._onCheckboxChange}">
+					${this.label}
+					<div slot="inline-help">${this.description}</div>
+					<d2l-expand-collapse-content slot="supporting" ?expanded="${this.checked}" aria-busy=${this._expandCollapseBusy ? 'true' : 'false'} @d2l-expand-collapse-content-expand="${this._onExpandCollapseContentToggled}" @d2l-expand-collapse-content-collapse="${this._onExpandCollapseContentToggled}">
+						<slot></slot>
+					</d2l-expand-collapse-content>
+				</d2l-input-checkbox>
 			`}
-			<d2l-input-checkbox-spacer class="d2l-input-checkbox-spacer">
-				<div class="d2l-input-checkbox-description d2l-skeletize">${this.description}</div>
```

Comment:
This "description" is styled to look like "body small", which is what the `inline-help` slot is for -- so I've moved that in there.

Reply:
Unfortunate that we need to keep the `color` and `font-size` properties around for the read-only version. 😢

Reply:
Yeah this made me a little sad, but also "labs".